<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TtbCharEventDialog</class>
 <widget class="QDialog" name="TtbCharEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit actor event</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>491</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Char:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="charEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>30</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="costumeEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transformation:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="eventLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Event:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="eventComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>68</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="customCheck">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Use custom audio/subtitles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen language:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="listenLangComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>108</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Japanese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="listenButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>108</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtitle language:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="subLangComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>148</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spanish (es)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spanish (ca)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>French</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>German</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Italian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Portuguese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Russian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese (tw)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese (zh)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Korean</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Japanese</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="subsEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TtbCharEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
